packages/integration-karma/helpers/test-utils.js

```diff
@@ -93,23 +93,28 @@ window.TestUtils = (function (lwc, jasmine, beforeAll) {
                         },
                     };
                 },
-                compare: function compare(actual, expectedMessage) {
-                    function matchMessage(message) {
+                compare: function compare(actual, expectedMessages) {
```

Comment:
I needed a way to test _multiple_ dev warnings, which is why I had to modify this.